pkg/api/vminfo.go

```diff
@@ -195,9 +196,9 @@ func ExtractVmInfo(vm *vmapi.VirtualMachine) (*VmInfo, error) {
 
 	// check: min <= using <= max
 	if using.HasFieldLessThan(min) {
-		return nil, fmt.Errorf("current usage %+v has field less than minimum %+v", using, min)
+		klog.Warningf("current usage %+v has field less than minimum %+v", using, min)
```

Comment:
With this change, we lose some context about which VM triggered the warning (before, it was the caller's responsibility to include it in the logs). It'd be good to add that into the message here. Maybe something along the lines of:

```go
klog.Warningf("VM %v current usage ...", info.NamespacedName(), ...)
```

(`NamespacedName()` returns type `NamespacedName`, which provides some standardized formatting for VM names)